test/functional/api/v2/hosts_controller_test.rb

```diff
@@ -100,6 +100,34 @@ def expect_attribute_modifier(modifier_class, args)
     assert_response :created
   end
 
+  test "should create host with default pxe_loader" do
```

Comment:
Code duplication, these tests are the same in hosts/hostgroups , move them to `shared`